### DISCUSSION TITLE
Duda parte 2 pregunta 5

### DISCUSSION THREAD

**Discussion by eayaach** [IIC2133-PUC/2024-2#18] [Tarea 0] [answered]
Con respecto al intervalo que nos entregan [start - end], si un pinguino quisiera por ejemplo regalar 5 puffles desde el inicio, ¿como deberia ser el input?: [0,4] o [1,5]. Y lo otro, ¿podemos asumir que el intervalo siempre pertenece a los puffles que tiene el pinguino?

**Comment by 4n1t4** [comment, marked as answer]
El primer índice es 0. Luego, respecto a la segunda pregunta, si te refieres a que si el pingüino que regale sus puffles efectivamente siempre tiene el intervalo pedido, esto siempre es así.
